app/components/AppUpdateNotification.tsx

```diff
@@ -0,0 +1,160 @@
+/* eslint-disable react/no-unused-prop-types */
+import React, {
+    useState,
+    useCallback,
+    useEffect,
+    useMemo,
+    Dispatch,
+    SetStateAction,
+    ComponentType,
+} from 'react';
+
+import CogIcon from '@resources/svg/settings.svg';
+import ErrorIcon from '@resources/svg/logo-error.svg';
+import CheckmarkIcon from '@resources/svg/logo-checkmark.svg';
+import Button from '~/cross-app-components/Button';
+import { NotificationLevel } from '~/features/NotificationSlice';
+import Loading from '~/cross-app-components/Loading';
+
+import Notification from './Notification';
+import Countdown from './Countdown';
+
+enum UpdateStatus {
+    Available,
+    Downloading,
+    Verifying,
+    Success,
+    Error,
+}
+interface Props {
+    version: string;
+    onUpdate(): void;
+    onClose(): void;
+}
+
+interface UpdateState {
+    status: UpdateStatus;
+    message?: string;
+}
+
+interface ChildProps extends Props, UpdateState {
+    setState: Dispatch<SetStateAction<UpdateState>>;
+}
+
+const AvailableChild = ({
+    onUpdate,
+    setState,
+    onClose,
+    version,
+}: ChildProps) => {
+    const update = useCallback(() => {
+        onUpdate();
+        setState({ status: UpdateStatus.Downloading });
+    }, [onUpdate, setState]);
+
+    return (
+        <>
+            <div className="flex alignCenter">
+                <CogIcon width="21" height="22" />
+                <span className="mL5">Version {version} is available.</span>
```

Comment:
In my testing this just shows as:
```
Version is available.
```
i.e. `{version}` seems to be undefined.

Reply:
should be fixed now.. thought I had checked this - sorry! 😅